mac/Keyman4Mac/Keyman4Mac/Frameworks/KeymanEngine4Mac.framework/Headers

```diff
@@ -0,0 +1 @@
+Versions/Current/Headers
```

Comment:
I did/do wonder about that. I tried deleting it, and Keyman4Mac builds both in Xcode and at the command line, but in Xcode there are a bunch of warnings about missing files. I can't seem to find where those missing files are referenced. Based on a commented-out import statement in AppDelegate.m, it looks like the original idea was to have the test app link in the engine framework, but then the test app wanted to access some header files that weren't made public by the framework, so instead the test app just included all the files (headers + source) from the engine and rebuilt them as part of the test app. The IM (by contrast) does use the framework (but doesn't check it into source control).